src/server/actions/Event.ts

```diff
@@ -4,7 +4,10 @@ import { CreateEventVolunteerRequest } from '@/types/dataModel/eventVolunteer';
 import EventVolunteer from '@/server/models/EventVolunteer';
 import EventSchema from '@/server/models/Event';
 import EventVolunteerSchema from '@/server/models/EventVolunteer';
-import { CreateEventRequest } from '@/types/dataModel/event';
+import {
```

Comment:
This file was already changed because of your last pull request so just merge main into this branch before so I dont have to resolve conflicts.